docs/hardware/devices/index.mdx

```diff
@@ -24,7 +24,7 @@ Please do your research and choose the board that meets your needs (or maybe alr
 
 :::info
 
-- Devices using the previous-generation Semtech SX1276 are no longer recommended due to their inferior performance.
+- We recommend devices that use the newer Semtech SX1262 due to improved performance. Avoid the previous-generation Semtech SX1276.
```

Comment:
```suggestion
- If possible, we recommend choosing devices equipped with the newer Semtech SX1262, as it offers improved performance compared to the SX1276.
```

Reply:
Maybe also change "SX1262" into "SX126x series" and "SX1276" into "SX127x series".

Reply:
updated the suggestion